web/react/utils/channel_intro_mssages.jsx

```diff
@@ -214,3 +173,15 @@ export function createStandardIntroMessage(channel, showInviteModal) { //eslint-
         </div>
     );
 }
+
+function createSetHeaderButton(channel) { // eslint-disable-line react/no-multi-comp
```

Comment:
eslint seems to think this is declaring another component for some reason so I had to disable the error message there